Forge.TreeWalker/src/TreeNodeContext.cs

```diff
@@ -0,0 +1,102 @@
+//-----------------------------------------------------------------------
+// <copyright file="TreeNodeContext.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// <summary>
+//     The TreeNodeContext class.
+// </summary>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Forge.TreeWalker
+{
+    using System;
+    using System.Threading;
```

Comment:
>using System.Threading; [](start = 4, length = 23)

I don't think this using statement is required either, please remove if possible.

Reply:
CancellationToken is from "System.Threading". Need to keep as it is.

---
In reply to: [506629130](https://github.com/microsoft/Forge/pull/32#discussion_r506629130) [](ancestors = 506629130)